### PR TITLE
Update README to be more complete and clear

### PR DESCRIPTION
**Pull request by air-kyi**
## What does this PR do?
Edit docs after read-through

* Make links accessible to screenreaders
* Add flexibility to instructions for various operating systems (Windows) and terminals (git bash)
* Fix typos

## How was the functionality tested and verified?
NA

### CODE REVIEW

**Comment by michplunkett** [comment]
Couple of things:
1. Could you write the title of the PR in the present imperative tense? Examples of that tense are: 
"Fix issue in the dispatcher where…", "Improve our handling of…", etc."
2. Could you write the PR description using the format that was already there? Leave the tested and verifying part tho.
```markdown
## What does this PR do?
[your description goes here]
```

File: CONTRIBv2.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Interested in writing a story with the Maroon Data Team? Follow the steps below.
 4. Download node which will allow you to run Javascript, the programming language we use to write our visualizations in:
     - [node.js](https://nodejs.org/en) `v20.9.0`
         - To install the correct version, open the terminal and run:
-            1. Install [Node Version Manager](https://github.com/nvm-sh/nvm) with `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash` 
+            1. Install [Node Version Manager](https://github.com/nvm-sh/nvm) with `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash` . If you are getting error of alias contains slash (likely if you are on Windows) try using `nvm-setup.exe` from the [Windows release site](https://github.com/coreybutler/nvm-windows/releases).
```
Comment (michplunkett) [inline CONTRIBv2.md]: `If you are getting error of alias contains slash` -> `If you are receiving the error "alias contains slash"`

Something like that.

File: CONTRIBv2.md [hunk 2]
```diff
@@ -37,27 +37,31 @@ Interested in writing a story with the Maroon Data Team? Follow the steps below.
...
-- We primarily use [`HighCharts`](https://www.highcharts.com/) a javascript library to build visualizations.
+
+- We primarily use [`HighCharts`](https://www.highcharts.com/), a JavaScript library, to build visualizations. Each HighChart graphic is embedded on a simple HTML page that is linked in its parent story.
     - The best place to start is to find a previously created visual from the Maroon Team by opening [example-graphs.html](./example-graphs.html) from your local repository. This will have example charts and links to the accompanying code to review. When creating a data visualization, finding a chart similar to what you’re trying to develop and working off the existing JavaScript code linked.
     - If there is not an example similar to what you want to create, review the [`Demos` tab](https://www.highcharts.com/demo/) on the Highcharts website and review the code. You can find the code for that specific visual under the `View Code` or `Copy JS code` tabs under the visualization.
-- Other tools includes:
+    - Continue to [Build Process for a Visualization](#build-process-for-a-visualization)
```
Comment (michplunkett) [inline CONTRIBv2.md:46]: Nice.

**Comment by michplunkett** [comment]
You should be submitting pull requests into `main`, not any other branch. I am also a little unsure as to why there is a README_edit and CONTRIBv2.

**Comment by air-kyi** [comment]
Hi Michael, first of all: Thanks for bringing me onto the Maroon, things are officially getting going as you can see! (I am Karen by the way.) You can also see that things are a work in progress...This branch is a draft-mode update to the docs written by Austin that I am testing out and giving feedback to. I'm also not sure why there's several versions but assuming it will be cleaned up before school year starts. And thanks for reviewing this PR as an alumnus, lol, lead devs are in the process of fixing the default tagged reviewers.

**Comment by air-kyi** [comment]
Comments addressed btw

**Review by asteinhart — APPROVED** [review]

**Pull request merged**